build/deploy/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM busybox
+FROM buildpack-deps:jessie-curl
```

Comment:
It doesn't matter much, but this could be `debian:jessie` (or equivalently `debian:latest`) since we just want the basics and don't need what buildpack-deps adds. Alpine linux is a popular choice for minimal docker images, but it may be a little too minimal for us (it's based on musl and go binaries don't seem to work there).